wp-includes/sqlite/class-wp-sqlite-pdo-engine.php

```diff
@@ -49,6 +50,11 @@ class WP_SQLite_PDO_Engine extends PDO { // phpcs:ignore
 	 */
 	public $pre_ordered_results = null;
 
+	/**
+	 * Class variable to store the last query.
+	 *
+	 * @var string
+	 */
 	public $last_translation;
 
```

Comment:
This one seems unused so we could remove it completely

```suggestion
```

Reply:
It's used in the `WP_SQLite_Crosscheck_DB` class, I assume we'll want to remove that class completely as well?

Reply:
Ah good spot! I'd keep it in the plugin – it's super useful for debugging discrepancies between MySQL and SQLite outputs, I don't think I'd get this far without it.